A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -984,16 +984,16 @@
                 </UseTemplate>
             </Component>
 
-            <Component ID="Brake_Fan">
+            <Component ID="Brake_Fan_Imported">
                 <UseTemplate Name="FBW_Push_Toggle">
-                    <NODE_ID>PUSH_BKRFAN</NODE_ID>
-                    <LEFT_SINGLE_CODE>0</LEFT_SINGLE_CODE>
-                    <SEQ1_CODE>0</SEQ1_CODE>
-                    <SEQ1_NODE_ID>PUSH_AUTOBKR_SEQ1</SEQ1_NODE_ID>
-                    <TOOLTIPID>(Inop.)</TOOLTIPID>
-                    <ANIMTIP_0>TT:COCKPIT.TOOLTIPS.APU_SWITCH_SET_OFF</ANIMTIP_0>
-                    <ANIMTIP_1>TT:COCKPIT.TOOLTIPS.APU_SWITCH_SET_ON</ANIMTIP_1>
-                    <CURSOR/>
+                    <NODE_ID>PUSH_BKRFAN_IMPORTED</NODE_ID>
+                    <ANIM_NAME>PUSH_BKRFAN_IMPORTED</ANIM_NAME>
+                    <SEQ1_NODE_ID>PUSH_BKRFAN_IMPORTED_SEQ1</SEQ1_NODE_ID>
+                    <SEQ2_NODE_ID>PUSH_BKRFAN_IMPORTED_SEQ2</SEQ2_NODE_ID>
+                    <SEQ1_CODE>(L:A32NX_BRAKES_HOT, Bool)</SEQ1_CODE>
+                    <SEQ2_CODE>(L:A32NX_BRAKE_FAN, Bool)</SEQ2_CODE>
+                    <TOGGLE_SIMVAR>L:A32NX_BRAKE_FAN_BTN_PRESSED</TOGGLE_SIMVAR>
+                    <TOOLTIPID>%((L:A32NX_BRAKE_FAN_BTN_PRESSED, Bool))%{if}Release the brake fan button%{else}Engage the brake fan button%{end}</TOOLTIPID>
```

Comment:
```suggestion
                    <TOOLTIPID>%((L:A32NX_BRAKE_FAN_BTN_PRESSED, Bool))%{if}Turn ON brake fan%{else}Turn OFF brake fan%{end}</TOOLTIPID>
```

Reply:
The brake fan isn't 100% tied to the button, for instance if the gear is up, the brake fan will not be running even if the button is pressed. So I think the text "Engage/release the brake fan button" is closer to what the button actually does

Reply:
I'm not really a fan of the current wording, is there anything else you can think of?

Also, you can't engage auto brake max in the air but it still says "Set auto-brake to MAX"